src/main/java/com/jcabi/jdbc/JdbcSession.java

```diff
@@ -106,6 +106,7 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 0.1.8
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```

Comment:
@amihaiemil along with that we should add puzzle to break down this class, as it became too big